files/waggle-dance-federation.yml

```diff
@@ -11,3 +11,6 @@ primary-meta-store:
   remote-meta-store-uris: thrift://localhost:9083
   writable-database-white-list:
   - default
+spring:
```

Comment:
why do you need to set this if the default is true here? https://github.com/ExpediaGroup/waggle-dance/blob/77961143554f16b16857eacb6f516e90cd7d852e/waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/WaggleDance.java#L70

We should need to set this kind of settings in the WD config files.

Reply:
Updated WD version that no longer required this config